homeassistant/components/hue/strings.json

```diff
@@ -97,13 +102,30 @@
     },
     "sensor": {
       "zigbee_connectivity": {
+        "name": "Zigbee Connectivity",
         "state": {
           "connected": "[%key:common::state::connected%]",
           "disconnected": "[%key:common::state::disconnected%]",
           "connectivity_issue": "Connectivity issue",
           "unidirectional_incoming": "Unidirectional incoming"
         }
       }
+    },
+    "switch": {
+      "motion_sensor_enabled": {
+        "name": "Motion sensor enabled",
+        "state": {
+          "on": "[%key:common::state::enabled%]",
+          "off": "[%key:common::state::disabled%]"
+        }
```

Comment:
I would go for Motion sensor -> enabled/disabled or Motion sensor enabled -> on/off

Reply:
Yeah, I was playing a bit with this and ended up with this, open for suggestions. I wanted to prevent the confusion with the actual motion sensor so decided on the "enabled" suffix to make it more clear this switch is for enabling/disabling the actual sensor feature. Not sure about the state though. You think we should drop the state translation and keep the default on/off here ?

Reply:
Otherwise I think it's a bit double. I haven't seen a lot of integrations with custom states for switches or binary sensors so I think that's fine

Reply:
Removed the custom states.